packages/x-components/src/x-modules/related-prompts/components/related-prompt.vue

```diff
@@ -64,39 +50,31 @@
         type: Object as PropType<RelatedPrompt>,
         required: true
       },
-      nextQueryButtonClass: {
-        type: String,
-        default: 'x-button-outlined'
+      arePromptsVisible: {
```

Comment:
In `RelatedPromptsTagList` we talk about all the prompts being visible (plural). In this `RelatedPrompt` is about a single prompt, so I think here the prop should be called `isPromptViible`

Reply:
You're right